### PR TITLE
hostif: add client signal rate attribute

### PR DESCRIPTION
**Pull request by skuwa**
I added  client signal rate attribute.
related: #76

### CODE REVIEW

**Comment by ishidawataru** [comment]
How about changing the prefix `TAI_HOST_INTERFACE_CLIENT_SIGNAL_RATE_` to `TAI_HOST_INTERFACE_SIGNAL_RATE_` ?

since host interfaces are always used as a client, having "client" in the prefix looks redundant.

**Comment by skuwa** [comment]
@ishidawataru 
Thank you for your comment. I think it is better.

File: inc/taihostif.h [hunk 1]
```diff
@@ -40,6 +40,17 @@ typedef enum _tai_host_interface_tx_align_status_t
     TAI_HOST_INTERFACE_TX_ALIGN_STATUS_DESKEW_LOCK
 } tai_host_interface_tx_align_status_t;
 
+/** @brief The host interface signal rate */
+typedef enum _tai_host_interface_signal_rate_t
+{
+    TAI_HOST_INTERFACE_SIGNAL_RATE_UNKNOWN,
+    TAI_HOST_INTERFACE_SIGNAL_RATE_100_GbE,
```
Comment (ishidawataru) [inline inc/taihostif.h]: nit: GbE to GBE to follow the convention

Comment (skuwa) [reply inc/taihostif.h]: I got it. I have done.

**Pull request merged**